core/src/lib.rs

```diff
@@ -4,14 +4,27 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+//! This package contains the core implementation of Citadel, a self-sovereign identity protocol for decentralized networks.
+//!
```

Comment:
I would maybe add a line saying that: SP stands for Service Provider and LP for License Provider. Otherwise, I am not sure if the acronyms will be clear to people not familiar with the protocol.